src/app/lib/config/services/zhongwen.js

```diff
@@ -243,6 +243,7 @@ export const service = {
         listen: '收听',
         watch: '观看',
         liveLabel: '直播',
+        nextLabel: 'NEXT',
```

Comment:
There should also be a label for the other zhongwen variant.

Reply:
Ah, thank you - I'll fix that now